packages/console-client/oas-schema/console-apis-schema.json

```diff
@@ -6,6 +6,137 @@
         "description": "Mia-Platform Console Project. "
     },
     "paths": {
+        "/api/userinfo": {
```

Comment:
Is this the actual exported schema from API Portal or did you have to tweak some properties around to make it work?

Reply:
(just asking, but i see nothing complex on these schemas so i'm guessing it's the actual one)